style.css

```diff
@@ -52,14 +77,15 @@
   }
 }
 @media (min-width: 1280px) {
-  .container {
+  .header, 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5482251)